tests/ui/impl-trait/in-trait/new-lowering-strategy/trait-object-impl-trait.rs

```diff
@@ -0,0 +1,34 @@
+//compile-flags: -Zlower-impl-trait-in-trait-to-assoc-ty
```

Comment:
Do we need a copy of this test? Why not just add this flag as a revision on the original test (you said you copied this, right?)

Reply:
My idea was to place a PR, to be merged before this one, where I just remove `tests/ui/impl-trait/in-trait/new-lowering-strategy` and I add revisions for all the existing tests that work with `-Zlower-impl-trait-in-trait-to-assoc-ty`. So let's leave this is WIP but we can focus on discuss if these filters are ok, or we need a different way to fix this.